enterprise/server/auth/auth.go

```diff
@@ -484,29 +485,13 @@ func NewOpenIDAuthenticator(ctx context.Context, env environment.Env, authConfig
 	return a, err
 }
 
-func sameHostname(urlStringA, urlStringB string) bool {
-	if urlA, err := url.Parse(urlStringA); err == nil {
-		if urlB, err := url.Parse(urlStringB); err == nil {
-			return urlA.Hostname() == urlB.Hostname()
-		}
-	}
-	return false
-}
-
 func (a *OpenIDAuthenticator) validateRedirectURL(redirectURL string) error {
-	if a.myURL.Host == "" {
-		return status.FailedPreconditionError("You must specify a build_buddy_url in your config to enable authentication. For more information, see: https://www.buildbuddy.io/docs/config-app")
-	}
```

Comment:
nit: do we want to keep this check? Seems to give a nicer error message if the user just forgot to specify build_buddy_url

Reply:
Done.